pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1875,7 +1875,11 @@ def buildScriptsAssemble(
                                 context.println '[CHECKOUT] Checking out to adoptium/temurin-build...'
                                 repoHandler.checkoutAdoptBuild(context)
                                 printGitRepoInfo()
-                                if ((buildConfig.TARGET_OS == 'mac' || buildConfig.TARGET_OS == 'windows') && buildConfig.JAVA_TO_BUILD != 'jdk8u' && enableSigner) {
+                                if ((buildConfig.TARGET_OS == 'mac' || buildConfig.TARGET_OS == 'windows') && (buildConfig.JAVA_TO_BUILD == 'jdk11u' ||
+                                                                                                               buildConfig.JAVA_TO_BUILD == 'jdk17u' ||
+                                                                                                               buildConfig.JAVA_TO_BUILD == 'jdk21u' ||
+                                                                                                               buildConfig.JAVA_TO_BUILD == 'jdk23u')
+                                                                                                               && enableSigner) {
```

Comment:
We need this dual pass build with signing to resolve: https://github.com/adoptium/temurin-build/issues/4057#issuecomment-2504002390
and https://github.com/adoptium/temurin-build/issues/3997
I shall be working on a fix for these

Reply:
Shall I close this PR?